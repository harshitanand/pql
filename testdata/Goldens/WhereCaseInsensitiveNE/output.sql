SELECT * FROM "StormEvents" WHERE lower("State") <> lower('georgia');
