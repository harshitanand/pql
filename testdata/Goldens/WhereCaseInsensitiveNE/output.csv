EventId,State,EventType,DamageProperty
11032,ATLANTIC SOUTH,Waterspout,0
11098,FLORIDA,Heavy Rain,0
60913,FLORIDA,Tornado,6200000
13913,MISSISSIPPI,Thunderstorm Wind,20000
